packages/tooling/fast-tooling-react/src/form/custom-controls/control.css.md

```diff
@@ -0,0 +1,122 @@
+# Abstract
+
+The CSS controls consist of an abstract control and two extended controls, inline CSS and internal CSS. These controls will provide a way to interpret CSS strings for inline and internal CSS into form UI and back to strings.
+
+## Build script
+
+The package should have a `devDependency` for [MDN data](https://www.npmjs.com/package/mdn-data) and re-interpret the properties based on this. To only require what is needed there should be a build script which takes the MDN data and converts it to a `.ts` file with exports conforming to an interface which defines the example provided in [Variables](#variables).
+
+## Variables
+
+There should be a list of internal variables that are assigned to each CSS property. The structure should look like this:
+
+```typescript
+[
+    {
+        "border": { // The property name should be pascalCase
+            name: "border", // The name should be spinal-case
+            default: "none",
+            value: [
+                {
+                    type: "width",
+                    default: "1px"
+                },
+                {
```

Comment:
Trying to follow this example - should this be "style" instead of "border-style" since none of the other border properties prefix with "border"?

Reply:
Good point, I'll change it, I think I left it in there accidentally.

Reply:
So I added a `mapToCSSProperty` to indicate the CSS property the value maps to, that is what I was trying to also capture.